component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/smsotp/SMSOTPAuthenticator.java

```diff
@@ -1522,4 +1540,33 @@ private void setUserClaimValues(AuthenticatedUser authenticatedUser, Map<String,
             throw new AuthenticationFailedException(errorMessage, e);
         }
     }
+
+    private String getUnlockTimeInMilliSeconds(AuthenticatedUser authenticatedUser)
```

Comment:
Can we implement this method to return a `long`?

Reply:
fixed in 23e2fc0